_rules/additional-content-triggered-on-hover-is-hoverable-ep1s13.md

```diff
@@ -28,13 +28,14 @@ The rule applies to any element that when [hovered][] causes [visible changes in
 
 ## Expectation
 
-There are no [visible changes in content][] while the target element is [hovered][] or the mouse pointer remains inside the [area affected by the changes][area affected by visible changes].
+There are no [visible changes in content][] in the [area affected by the changes][area affected by visible changes] while the target element is [hovered][] or the mouse pointer remains inside that area.
```

Comment:
I thought about this solution too. But a visual indication on the menuitem that a submenu underneath that menuitem has been opened is rather common in these situations right?

Is you have 

Contact us V

If you hover over that you get

Contact us >
Via phone V
Via email V
Via fax V

And then you hover over email and you get 

Contact us >
- Via phone V
- Via email >
- - Our main branche
- - Our secondary brance
- - Our tertiary brance
- Via fax V

This would still fail this rule right? Because the area is definitely effected, even if the sub-sub menu doesn't open in the middle of the sub menu but off to the side, the little arrow denoting that "email" has been opened is still changed pixels right?

Can we write this rule so content is not allowed to disappear? Only changed or added to?

Reply:
Yes, that example would fail this rule (that is why I've added the last assumption when I made this change).

I couldn't find a way to make this rule work with distinctions about modifying content (disappearing, changing, adding). The problem is how you objectively identify that. This is why "visible changes in content" considers changes in rendered pixels and not changes of the content or properties of elements.

For example: if to change what is rendered on a page you delete node X that is a child of node Y, then create node Z and append it to node Y, from the user perspective you changed content, but from the DOM's perspective you removed and added content. In this example, content disappeared, which would fail your formulation. 

Would the existing assumption work for you? Or do you find it too strong?